Upgrade.md

```diff
@@ -0,0 +1,64 @@
+# Upgrading Substrate Dependencies
+
+This guide will hopefully help upgrade Substrate in the future.
+
+This is a guide based on updating from `0.9.25` to `0.9.30` and the issues it caused can be seen [here](https://github.com/digicatapult/dscp-node/pull/91/files)
+
+# First Thing
+
+Create an **integration** branch which all other branches will be PR'd into. This reduces git issues like having to use `git reset`.
+
+Upgrade the references in the repo. By doing this it will ensure that the `cargo build` (as seen below) works and the rest of the process can take place.
```

Comment:
feel like there could be more explanation what 'upgrade the references' means

Reply:
> upgrade the references to the `paritytech/substrate` repo in each of the wrokspaces `cargo.toml` files

?